book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental unless @rentals.include?(rental)
```

Comment:
- [ ] According to the project requirements and relationship between the `Person` and `Book` calsses using the intermediate class `Rental`, I think it would be better if your update your `#add_rental` method implementation as it should include a `date` and the `person` as parameter and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. i.e.

```ruby
  require_relative 'rental'
  

  ...
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```